lang/cs/Org.Apache.REEF.Driver/packages.config

```diff
@@ -18,6 +18,8 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Hadoop.Avro" version="1.5.6" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
```

Comment:
should be <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />